Gems/PhysX/Code/Source/RigidBodyComponent.cpp

```diff
@@ -309,6 +320,21 @@ namespace PhysX
         AzPhysics::SimulatedBodyComponentRequestsBus::Handler::BusConnect(GetEntityId());
     }
 
+    void RigidBodyComponent::ApplyPhysxSpecificConfiguration()
+    {
+        if (auto* body = GetRigidBody())
+        {
+            if (auto* pxRigidDynamic = static_cast<physx::PxRigidDynamic*>(body->GetNativePointer()))
+            {
```

Comment:
suggestion/nit: Could use early returns here instead to avoid as much indentation (not a huge deal though)